### ISSUE TITLE
Hover effect should only appear when a grabbable object is in focus on desktop

### ISSUE THREAD

**Issue by misslivirose** [mozilla/hubs#1638]
**Description*
Hover effect appears on grabbable objects / spawners even when the spawner is not targeted with the pointer. The video linked below shows how being in pause mode makes this effect show up and captured from the camera in-world, which I think it not desirable. 

**To Reproduce**
1. Be in a room with grabbable media or a spawner
2. Enter pause mode

**Expected behavior**
I don't think I'd expect the effect to show up in pause mode unless I was targeting the item, but I can see utility in it exposing the interactables within a scene. Ideally, though, I think we'd want to avoid having it show up in the camera preview window at a minimum.

**Screenshots**
https://docs.google.com/document/d/1wOtKzGvWnaaMp4Xcx8ykmVujo-r7qwzjN5aPlwyRSmI/edit?usp=sharing

**Comment by joshmarinacci** [comment]
Has this been fixed?

**Comment by misslivirose** [comment]
Nope, still reproduces. Part of a bigger discussion about surfacing interactions in pause mode, I think
